src/Sulu/Bundle/AdminBundle/Resources/js/views/Preview/Preview.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import React from 'react';
+import {observer} from 'mobx-react';
+import {sidebarStore} from '../../containers/Sidebar';
+import Icon from '../../components/Icon/Icon';
+
+type Props = {};
+
+@observer
+export default class Preview extends React.Component<Props> {
+    handleToggleSidebarClick = () => {
+        if (sidebarStore.size === 'medium') {
+            return sidebarStore.setSize('large');
```

Comment:
Do you need the return value here? Doesn't look like that, and just to be consistent with the other code path I would go for something like this:

```javascript
sidebarStore.setSize('large');
return;
```